src/webgl/light.js

```diff
@@ -315,4 +315,35 @@ p5.prototype.lights = function() {
   return this;
 };
 
+/**
+ * @method fallOff
+ * @chainable
+ */
+p5.prototype.fallOff = function(constAtt, linearAtt, quadAtt) {
+  this._assert3d('fallOff');
+  // p5._validateParameters('fallOff', arguments);
+
+  if (constAtt < 0) {
+    constAtt = 0;
+  }
+
+  if (linearAtt < 0) {
+    linearAtt = 0;
+  }
+
+  if (quadAtt < 0) {
+    quadAtt = 0;
+  }
+
+  if (constAtt === 0 && (linearAtt === 0 && quadAtt === 0)) {
```

Comment:
Are you doing this to prevent the attenuation from causing divide by zero problems in the shader? Maybe it should be made clear to the user why they cannot set all falloff values to zero by throwing a warning in the console letting them know that constantAtt has been set to 1.

Reply:
I'll log the console.warn with every check.